docs/csharp/deconstruct.md

```diff
@@ -60,7 +59,7 @@ The following example illustrates the use of tuples with discards. The `QueryCit
 
 ## Deconstructing user-defined types
 
-C# does not offer built-in support for deconstructing non-tuple types. However, as the author of a class, a struct, or an interface, you can allow instances of the type to be deconstructed by implementing one or more `Deconstruct` methods. The method returns void, and each value to be deconstructed is indicated by an [out](language-reference/keywords/out-parameter-modifier.md) parameter in the method signature. For example, the following `Deconstruct` method of a `Person` class returns the first, middle, and last name:
+C# does not offer built-in support for deconstructing non-tuple types other than [`record` types](#deconstructing-a-record-type). However, as the author of a class, a struct, or an interface, you can allow instances of the type to be deconstructed by implementing one or more `Deconstruct` methods. The method returns void, and each value to be deconstructed is indicated by an [out](language-reference/keywords/out-parameter-modifier.md) parameter in the method signature. For example, the following `Deconstruct` method of a `Person` class returns the first, middle, and last name:
```

Comment:
The only other built-in support is for [`KeyValuePair<TKey, TValue>`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Collections/DictionaryEntry.cs#L38-L42):

```csharp
foreach (var (key, value) in new Dictionary<int, string>
         {
             [1] = "One", [2] = "Two", [3] = "Three"
         })
{
    Console.WriteLine($"{key} = {value}");
}
```

I knew about the KVP, but did a [quick search for other options](https://github.com/dotnet/runtime/search?q=Deconstruct).

Reply:
Good catch, thanks!